jobs/utilityapi_software_engineer.html

```diff
@@ -8,7 +8,7 @@
     name: Daniel Roesler
```

Comment:
Also, I missed this in the previous review, but can you put a more precise location in the Location field - unless this is a 100% remote working job we want to see at least the city & state.

Reply:
Yes, this is a 100% remote working job in the United States. Should I keep it as United States or put our central office (Oakland, CA) even if they don't have to work there?